hack/testing/init-log-stack

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 OS_ANSIBLE_VERBOSITY=${OS_ANSIBLE_VERBOSITY:-"vv"}
-OS_VERSION_STRING=${OS_VERSION:+openshift_release=$OS_VERSION}
+OS_VERSION_STRING="openshift_release=v1.5.0"
```

Comment:
The default is that `OS_VERSION` is not set at all.  openshift is already installed on the system.  ansible should just figure out (facts, etc.) what `openshift_release` is already installed.